### DISCUSSION TITLE
Version 8 Plan

### DISCUSSION THREAD

**Discussion by rober72hv** [TerriaJS/terriajs#5134] [General] [answered]
Hello, happy 2021!!
I have seen that the next branch has not moved for 25 days.
Was there a branch change for version 8?
Any change in project planning?
Best
Rob

**Comment by AnaBelgun** [comment, marked as answer]
Hi @rober72hv and Happy New Year to you too! We're slowly coming back from leave and resuming development on the next branch. We already have a couple of PRs under review. The cadence of v8 releases will again be monthly. Kind regards,
